templates/layout.html

```diff
@@ -95,9 +95,6 @@
                                 <li><a href="{{ donation_link }}" target="_BLANK">Donate</a></li>
                                 <li class="divider"></li>
                                 <li><a href="/random">Random Mod</a></li>
-                                <li><a href="mumble://spacedock.info/SpaceDock?title=Lobby&version=1.2.0">Mumble</a></li>
-                                <li><a href="{{ irc_channel }}" target="_blank">Chat</a></li>
-                                <li><a href="/blog">Blog</a></li>
```

Comment:
Let's keep the blog. It's not hurting anyone and could be useful for announcements.